### PR TITLE
:bug: Add tooltips for Priority and Preemption columns

### PR DESCRIPTION
**Pull request by rszwajko**
Resolves: https://issues.redhat.com/browse/MTA-3235

### CODE REVIEW

File: client/src/app/pages/tasks/tasks-page.tsx [hunk 1]
```diff
@@ -189,6 +190,11 @@ export const TasksPage: React.FC = () => {
     columnState,
   } = tableControls;
 
+  const tooltips: Record<string, ThInfoType> = {
+    priority: { tooltip: t("tooltip.priority") },
+    preemption: { tooltip: t("tooltip.preemption") },
+  };
```
Comment (sjd78) [inline client/src/app/pages/tasks/tasks-page.tsx:196]: The tooltip keys you added are at `wizard.tooltip.*`

Probably need to create a top level "tooltip" section, or move it to the "message" section.

File: client/src/app/pages/tasks/tasks-page.tsx [hunk 2]
```diff
@@ -189,6 +190,11 @@ export const TasksPage: React.FC = () => {
     columnState,
   } = tableControls;
 
+  const tooltips: Record<string, ThInfoType> = {
```
Comment (sjd78) [inline client/src/app/pages/tasks/tasks-page.tsx:193]:
```suggestion
  const tooltips: Record<string, ThProps["info"]> = {
```

It's easy to import `ThProps` and then reference the info prop.  Then there are no oddball imports required.

**Review by sjd78 — CHANGES_REQUESTED** [review]

**Review by sjd78 — APPROVED** [review]
LGTM

**Comment by rszwajko** [comment]
Screenshot from 2024-07-11 17-07-10
Screenshot from 2024-07-11 17-06-46

File: client/public/locales/en/translation.json [hunk 3]
```diff
@@ -510,6 +510,10 @@
       "save": "Failed to save {{type}}."
     }
   },
+  "tooltip": {
+    "priority": "Tasks priority(Low-High). Impacts tasks scheduling policy.",
```
Comment (mguetta1) [inline client/public/locales/en/translation.json:514]: Hi, I am not sure about this description regarding the priority value.The values can range from zero to any positive whole number.
`Low-High` doesn't say much to me. wdyt?

Comment (mguetta1) [reply client/public/locales/en/translation.json:514]: Suggestion: Please remove `(Low-High)`. The user can check the release notes documentation that says: `Priority: Priority of the task. The value is from zero to any positive integer. The higher the value in this column, the higher the priority of the task.`

Or use something like: `Tasks priority, 0 to any positive integer (Low-High). Impacts tasks scheduling policy.` as the tooltip

Comment (sjd78) [reply client/public/locales/en/translation.json:514]: Change made in #2028

**Pull request merged**